.gitignore

```diff
@@ -29,6 +29,7 @@ yarn-error.log*
 .idea/
 backfills/*.json
 .envrc
+.env.prod
```

Comment:
package.json references this, lets add it to gitignore by default. Leaving `.env.production` as some folks may have set that up already instead